features/step_definitions/viewing_browse_steps.rb

```diff
@@ -115,28 +113,25 @@ def top_level_browse_pages
 end
 
 def add_browse_pages
-  content_store_has_item '/browse', { links: {
+  content_store_has_item '/browse', links: {
     top_level_browse_pages: top_level_browse_pages
-  }}
+  }
 end
 
-def add_first_level_browse_pages(child_pages: second_level_browse_pages, order_type: order_type)
```

Comment:
I have to admit, I'm not sure what this was originally trying to do, because neither `second_level_browse_pages` nor `order_type` were defined. But none of the tests were relying on the default values, so deleting them seemed to be the safest thing to do.